app/models/repository_row.rb

```diff
@@ -0,0 +1,14 @@
+class RepositoryRow < ActiveRecord::Base
+  belongs_to :repository
+  belongs_to :team
```

Comment:
It might help a bit in some queries, but I don't think it is 100% necessary here, I will remove this association